web/src/app/util/Markdown.js

```diff
@@ -2,14 +2,63 @@ import React from 'react'
 import ReactMarkdown from 'react-markdown'
 import { safeURL } from './safeURL'
 import gfm from 'remark-gfm'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  markdown: {
+    '& h1, h2, h3, h4, h5, h6, p': {
+      marginTop: 0,
+      marginBottom: '0.5rem',
+    },
+    '& hr': {
+      margin: '1.5rem 0',
+    },
+    '& th': {
+      textAlign: 'left',
+      paddingRight: '8px',
+    },
+    '& td': {
+      paddingRight: '8px',
+    },
+    '& pre': {
+      display: 'block',
+      padding: '9.5px',
+      margin: '0 0 10px',
+      fontSize: '13px',
+      lineHeight: '1.42857143',
+      color: '#333',
+      wordBreak: 'break-all',
+      wordWrap: 'break-word',
+      backgroundColor: '#f5f5f5',
+      border: '1px solid #ccc',
+      borderRadius: '4px',
+    },
+    '& code': {
+      padding: '2px 4px',
```

Comment:
are the CSS properties from the global file used here? From the classname on [L12](https://github.com/target/goalert/pull/1290/files#diff-3a39745961e924d7b6600b2aa05f7daaef4304372542dfa5aa9c8cfd6930154bL12) previously it looks like only the `.react-markdown td` and `.react-markdown th` styles would be used for this component

Reply:
That makes sense, thanks for the clarification!